index.html

```diff
@@ -99,4 +96,4 @@ <h1 class='title'>hugo findlay's web site</h1>
 
     document.getElementById('back').innerText = planetarium;
 
-</script>
\ No newline at end of file
+</script>
```

Comment:
unrelated comment / lecture but in HTML `<script>` and `<style>` tags have gotta go in either `<head>` or `<body>`.

if you put them loose then the parser will just rewrite your HTML to put them inside `<body>`.

you can see this if you load this page in dev tools, the script and style are inside the `<body>`.